modules/react/button/lib/HyperlinkExternal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {colors} from '@workday/canvas-kit-react/tokens';
+import {
+  styled,
+  createComponent,
+  StyledType,
+  ElementComponent,
+} from '@workday/canvas-kit-react/common';
+import {HStack} from '@workday/canvas-kit-labs-react/layout';
+import {extLinkIcon} from '@workday/canvas-system-icons-web';
+import {SystemIcon, systemIconStyles} from '@workday/canvas-kit-react/icon';
+import {Hyperlink, HyperlinkProps} from './Hyperlink';
+
+export interface HyperlinkExternalProps extends HyperlinkProps {
+  /**
+   * Override the aria-label for the icon
+   * @default 'Opens link in new window'
+   */
```

Comment:
We should describe the purpose of this prop. It is meant to inform the user that the link opens in a new window and thus should be translated.

```suggestion
  /**
   * Informs a screen reader user the link will open in a new window. It is read after the link text.
   * This value will need to be translated.
   * @default 'Opens link in new window'
   */
```